tests/run_tests.sh

```diff
@@ -144,6 +144,9 @@ function setup_test_options()
             fi
         done
     else
```

Comment:
can you add a warning after this `else` condition, to avoid user input both TEST_CASES_FILE and TEST_CASES?